sdk/tables/Microsoft.Azure.WebJobs.Extensions.Tables/tests/WebJobsShared/TestHelpers.cs

```diff
@@ -145,148 +21,13 @@ public static IHostBuilder ConfigureDefaultTestHost(this IHostBuilder builder, A
                 })
                 .ConfigureTestLogger();
         }
-        public static IHostBuilder ConfigureDefaultTestHost<TProgram>(this IHostBuilder builder,
-           TProgram instance, Action<IWebJobsBuilder> configureWebJobs)
-        {
-            return builder.ConfigureDefaultTestHost(configureWebJobs, typeof(TProgram))
-                .ConfigureServices(services =>
-                {
-                    services.AddSingleton<IJobHost, JobHost<TProgram>>();
-                    services.AddSingleton<IJobActivator>(new FakeActivator(instance));
-                });
-        }
-        public static IHostBuilder ConfigureDefaultTestHost<TProgram>(this IHostBuilder builder)
-        {
-            return builder.ConfigureDefaultTestHost(o => { }, typeof(TProgram))
-                .ConfigureServices(services =>
-                {
-                    services.AddSingleton<IJobHost, JobHost<TProgram>>();
-                });
-        }
-        public static IHostBuilder ConfigureDefaultTestHost<TProgram>(this IHostBuilder builder, Action<IWebJobsBuilder> configureWebJobs,
-            INameResolver nameResolver = null, IJobActivator activator = null)
-        {
-            return builder.ConfigureDefaultTestHost(configureWebJobs, typeof(TProgram))
-                .ConfigureServices(services =>
-                {
-                    services.AddSingleton<IJobHost, JobHost<TProgram>>();
-                    if (nameResolver != null)
-                    {
-                        services.AddSingleton<INameResolver>(nameResolver);
-                    }
-                    if (activator != null)
-                    {
-                        services.AddSingleton<IJobActivator>(activator);
-                    }
-                });
-        }
+
         public static IHostBuilder ConfigureTestLogger(this IHostBuilder builder)
```

Comment:
Don't we already have this in shared source?

Reply:
Yep, I'll switch to it in the next PR.